apiserver/common/networkingcommon/subnets.go

```diff
@@ -55,7 +55,7 @@ func NewAddSubnetsCache(api NetworkBacking) *addSubnetsCache {
 // up in the cache (or populates the cache if empty).
 func (cache *addSubnetsCache) validateSpace(spaceTag string) (*names.SpaceTag, error) {
 	if spaceTag == "" {
-		return nil, errors.Errorf("SpaceTag is required")
+		return nil, nil
```

Comment:
Would be great if "" were a valid space name.
That would require a change to the names.v2 package though. But it would avoid all the special case code in this PR.